html/browsers/windows/noreferrer.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>rel=noreferrer nullifies window.opener</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  async_test(function(t) {
+    localStorage.clear()
+
+    var hyperlink = document.body.appendChild(document.createElement("a"))
+    hyperlink.rel = "noreferrer"
+    hyperlink.target = "_blank"
+    hyperlink.href = "support-opener-null.html"
+    hyperlink.click()
+    document.body.removeChild(hyperlink)
+
+    onstorage = function(e) {
+      t.step(function() {
+        assert_equals("null", e.newValue)
```

Comment:
I think the expectations are reversed here (e.g. it should be actual, then expected).